### PR TITLE
Adds a follow flag with a sleep config option

### PR DESCRIPTION
**Pull request by chentex**
New flags for the tool

```
  -f    Follow: do not exit after checking once, keep checking every N seconds.                                                                                                                                                               
        Use ctrl + c to exit.                                                                                                                                                                                                                 
  -sleep int
        Time in Seconds to sleep before next check. (default 30)
```

You can now call the tool like this

`./rlgl -f -sleep 5`

This will keep the tool running and checking every 5 seconds

Signed-off-by: Vicente Zepeda Mas <vzepedam@redhat.com>

### CODE REVIEW

File: rlgl.go [hunk 1]
```diff
@@ -29,10 +34,18 @@ func main() {
...
+		if CheckEvents(coreclient) && CheckNodeEvents(coreclient, t) {
+			os.Exit(0)
+		} else {
+			os.Exit(1)
+		}
+	}
+	for *f {
+		CheckEvents(coreclient)
+		CheckNodeEvents(coreclient, t)
+		time.Sleep(time.Duration(*sleep) * time.Second)
```
Comment (jtaleric) [inline rlgl.go:40]: wonder if the last state (when the ctrl+c is pressed) should be our exit code?

**Review by jtaleric — COMMENTED** [review]
LGTM, one thought, but it sort of doesn't make sense to report a exit code if the user just follows... 

Honestly, I just use watch `rlgl`

File: rlgl.go (hunk 1 above)
Comment (jtaleric) [reply rlgl.go:40]: or @chentex maybe we watch until an error occurs (if it occurs)... Thoughts?

Comment (chentex) [reply rlgl.go:40]: That could be a good option, catching the last status before ctrl + c might be tricky, but for sure we can exit(1) once we catch an error.

Comment (chentex) [reply rlgl.go:40]: Updated with the changes

**Review by jtaleric — APPROVED** [review]
nice!

**Pull request merged**